Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -327,12 +369,12 @@ void CGPathMoveToPoint(CGMutablePathRef path, const CGAffineTransform* m, float
 
     if (pathObj->_count + 1 >= pathObj->_max) {
```

Comment:
This entire code block is repeated in a few places, might make sense to move this to a utility function and call it instead.  Understood that this was in the existing code and not new, would still be a good improvement.